osu.Game/Online/Leaderboards/LeaderboardScore.cs

```diff
@@ -87,6 +87,9 @@ public partial class LeaderboardScore : OsuClickableContainer, IHasContextMenu,
         [Resolved]
         private ScoreManager scoreManager { get; set; } = null!;
 
+        [Resolved]
+        private OsuGame game { get; set; }
```

Comment:
```suggestion
        private OsuGame? game { get; set; }
```

to fix your issue. if a resolved field is annotated as nullable, it won't throw even if the dependency isn't registered

Reply:
Thanks I was trying that earlier and I was seeing some weird warning locally, but if it works on remote then that is problem solved.